command/aarelayer/service/aa_server_test.go

```diff
@@ -22,7 +24,12 @@ const baseURL = "127.0.0.1:8289"
 func Test_AAServer(t *testing.T) {
 	t.Parallel()
 
-	aaServer := getServer(t)
+	dbpath, err := os.MkdirTemp("/tmp", "aa_server_state_db")
```

Comment:
```suggestion
	dbpath, err := os.MkdirTemp("", "aa_server_state_db")
```

Reply:
go defines temp dir if it's empty(for different OS). Or you can use t.TempDir()

Reply:
fixed https://github.com/0xPolygon/polygon-edge/pull/1261/commits/32968985358fa9c2bba8d9bb27dc53966fe018ef